java/arcs/core/entity/Entity.kt

```diff
@@ -42,7 +42,7 @@ interface Entity : Storable {
         ttl: Ttl = Ttl.Infinite()
     )
 
-    fun serialize(): RawEntity
+    fun serialize(storeSchema: Schema? = null): RawEntity
```

Comment:
It would probably be a good idea to add a short KDoc here to explain the purpose of the new parameter, and expected behavior when it is/is not included.

Reply:
thanks, added kdoc